website/content/docs/secrets/rabbitmq.mdx

```diff
@@ -68,6 +68,25 @@ management tool.
     more about [RabbitMQ management tags][rmq-perms]
     and [RabbitMQ topic authorization][rmq-topics].
 
+Similarly, if a static username is required, one can configure a static role
+instead:
+
+1.  Configure a role that maps a name in Vault to virtual host permissions:
```

Comment:
Suggest folding the new instructions into the existing step as an alternative rather than adding a second list with only one step.

```suggestion
  Alternatively, if you require a static username, configure a static role
  that maps a name in Vault to virtual host permissions:
```

style correction: active voice, speak directly to the reader with "you",